cfe_internal/enterprise/ha/ha_update.cf

```diff
@@ -0,0 +1,129 @@
+bundle agent ha_update
+{
+ vars:
+      "master_hub_location" string => "/var/cfengine/state/master_hub.dat";
+      "hubs_keys_location" string => "/var/cfengine/ppkeys_hubs"; 
+      "ppkeys_staging" string => "/var/cfengine/ppkeys_staging";
+ methods:
+  enable_cfengine_enterprise_hub_ha.am_policy_hub::
+      "share_keys" usebundle => ha_share_hub_keys;
+      "sync_hub_data" usebundle => ha_hub_sync;
+
+  enable_cfengine_enterprise_hub_ha::
+      "sync_agent_data" usebundle => ha_agent_sync;
+
+  enable_cfengine_enterprise_hub_ha.hub_data_synced::
+      "manage_keys" usebundle => manage_hub_synced_data;
+}
+
+bundle agent ha_agent_sync
+{
+ files:
+  !am_policy_hub::
+    "$(sys.workdir)/policy_server.dat"
+      copy_from => ha_no_backup_scp("$(ha_update.master_hub_location)", @(update_def.policy_servers)),
+      comment => "Update master hub IP on CFEngine node",
```

Comment:
This ensures that remote agents are looking at the active policy hub right?

Maybe the comment can be improved ....
So that remote agents  continue to get policy we need to make sure that they look to the currently active hub as their policy server.